### ISSUE TITLE
Fallo en la notificacion desde redsys 303

### ISSUE THREAD

**Issue by Dabrilla** [DevopensourceTeam/Devopensource_Redsys#71]
Hola. Desde hace unos meses tengo un problema con el módulo, que ha venido funcionando bien mucho tiempo. Resulta que redsys no devuelve la función callback y el pedido se queda como pendiente. Sé que a mucha gente le ha pasado, pero no he encontrado a nadie que reporte el error que me da a mi de 303 en la notificación.
Si accedo a la url del callback (https) desde navegador, me redirecciona a la main, con lo que entiendo que es accesible. 

Gracias de antebraso.

**Comment by datelpiro** [comment]
Suerte con eso. Yo lo he dejado por imposible ya. Afortunadamente estoy migrando a Magento 2. Y olvídate de llamar al soporte de Redsys que si no usas su módulo no van a hacer nada.

**Comment by Dabrilla** [comment]
Sí, también estoy a medio de migrar, pero puede llevarme algunos meses, y mientras me gustaría tener esto resuelto porque es un coñazo.

**Comment by JustDevZero** [comment]
A mi me esta pasando lo mismo, en un proyecto que no es ni magento 1.7, ni magento. La ayuda que proporcionan desde Redsys es nula, cuando se les ha demostrado que no llega ni al servidor y que ese 303 se lo estan inventando...
